nominees/formpack.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Formpack",
+  "description": "Formpack is a tool used by kpi to parse and repackage survey data for exports, reports, and maps based on different languages available from the form definition.",
```

Comment:
@rachita19082 could you kindly spell out the abbreviation of `kpi` for readers to easily comprehend what this project is all about without having to use another search engine. Thanks.

Reply:
Resolved in commit 39d87b9